.github/build/friendly-filenames.json

```diff
@@ -0,0 +1,33 @@
+{
+    "android-arm64": { "friendlyName": "android-arm64-v8a" },
+    "darwin-amd64": { "friendlyName": "macos-64" },
+    "darwin-arm64": { "friendlyName": "macos-arm64-v8a" },
+    "dragonfly-amd64": { "friendlyName": "dragonfly-64" },
+    "freebsd-386": { "friendlyName": "freebsd-32" },
+    "freebsd-amd64": { "friendlyName": "freebsd-64" },
+    "freebsd-arm64": { "friendlyName": "freebsd-arm64-v8a" },
+    "freebsd-arm7": { "friendlyName": "freebsd-arm32-v7a" },
+    "linux-386": { "friendlyName": "linux-32" },
+    "linux-amd64": { "friendlyName": "linux-64" },
+    "linux-arm5": { "friendlyName": "linux-arm32-v5" },
+    "linux-arm64": { "friendlyName": "linux-arm64-v8a" },
+    "linux-arm6": { "friendlyName": "linux-arm32-v6" },
+    "linux-arm7": { "friendlyName": "linux-arm32-v7a" },
+    "linux-mips64le": { "friendlyName": "linux-mips64le" },
+    "linux-mips64": { "friendlyName": "linux-mips64" },
+    "linux-mipslesoftfloat": { "friendlyName": "linux-mips32le-softfloat" },
+    "linux-mipsle": { "friendlyName": "linux-mips32le" },
+    "linux-mipssoftfloat": { "friendlyName": "linux-mips32-softfloat" },
+    "linux-mips": { "friendlyName": "linux-mips32" },
+    "linux-ppc64le": { "friendlyName": "linux-ppc64le" },
+    "linux-ppc64": { "friendlyName": "linux-ppc64" },
+    "linux-riscv64": { "friendlyName": "linux-riscv64" },
+    "linux-s390x": { "friendlyName": "linux-s390x" },
+    "openbsd-386": { "friendlyName": "openbsd-32" },
+    "openbsd-amd64": { "friendlyName": "openbsd-64" },
+    "openbsd-arm64": { "friendlyName": "openbsd-arm64-v8a" },
+    "openbsd-arm7": { "friendlyName": "openbsd-arm32-v7a" },
+    "windows-386": { "friendlyName": "windows-32" },
+    "windows-amd64": { "friendlyName": "windows-64" },
```

Comment:
this should be `windows-amd64` in order to not change the format from previous release